python/ray/tune/tune.py

```diff
@@ -32,12 +35,34 @@ def _make_scheduler(args):
             args.scheduler, _SCHEDULERS.keys()))
 
 
+def _find_checkpoint_dir(exp_list):
+    if exp_list:
+        exp = exp_list[0]
+        # TODO(rliaw): Make sure this is resolved earlier.
+        return os.path.join(exp.spec["local_dir"], exp.name)
+    else:
+        return None
+
+
+def try_restore_runner(checkpoint_dir, search_alg, scheduler, trial_executor):
+    logger.warn("Restoring from previous experiment and "
```

Comment:
Remove this log (merge any warning with the other one about it being experimental).